src/main/java/com/quantumsoft/qupathcloud/converter/dicomizer/ImageToWsiDcmConverter.java

```diff
@@ -0,0 +1,116 @@
+package com.quantumsoft.qupathcloud.converter.dicomizer;
+
+import com.quantumsoft.qupathcloud.exception.QuPathCloudException;
+import org.apache.commons.io.FileUtils;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.openslide.OpenSlide;
+
+import java.io.File;
+import java.io.IOException;
+import java.text.MessageFormat;
+import java.util.Map;
+
+public class ImageToWsiDcmConverter {
+    private static final long MAX_TOP_SIZE = 2048;
+    private static final int MAX_TILE_SIZE = 512;
+    private static final int MIN_TILE_SIZE = 64;
+    private static final int DEFAULT_TILE_SIZE = 256;
+
+    private static final Logger LOGGER = LogManager.getLogger();
+    private File inputFile;
+    private File outputDirectory;
+
+    public ImageToWsiDcmConverter(File inputFile, File outputDirectory) {
+        this.inputFile = inputFile;
+        this.outputDirectory = outputDirectory;
+    }
+
+    public void convertImageToWsiDcm(String imageName) throws QuPathCloudException {
+        try {
+            Dicomizer.Options options = new Dicomizer.Options()
+                    .inputPath(inputFile.getPath())
+                    .outputFolder(outputDirectory.getPath())
+                    .imageName(imageName);
+            configurePyramidParameters(options);
+
+            Dicomizer dicomizer = new Dicomizer();
+            dicomizer.run(options);
+        } catch (Throwable e) {
+            LOGGER.error("Error converting image to dcm!", e);
+            throw new QuPathCloudException(e);
+        }
+    }
+
+    private String getOpenslidePath() throws IOException {
```

Comment:
This method doesn't seem to be used?

Reply:
Yeah, this and one below are legacy from using orthank-wsi dicomizer, will remove.